docs/faq.rst

```diff
@@ -158,8 +163,12 @@ For custom emoji, you should pass an instance of :class:`Emoji`. You can also pa
 can use said emoji, you should be able to use :meth:`Client.get_emoji` to get an emoji via ID or use :func:`utils.find`/
 :func:`utils.get` on :attr:`Client.emojis` or :attr:`Guild.emojis` collections.
 
+The name and ID of a custom emoji can be found with the client by prefixing :emoji: with a backslash.
```

Comment:
`:emoji:` has special meanings in rST.

Reply:
This should be resolved, and clarified I think with ` ``:custom_emoji:`` `